UI/NativeFrameNav/NativeFrameNav.Shared/BlankPage2.xaml

```diff
@@ -15,11 +16,12 @@
 			<RowDefinition Height="*" />
 		</Grid.RowDefinitions>
 
-		<CommandBar Grid.Row="0" Content="BlankPage2">
-			<CommandBar.PrimaryCommands>
-				<skia:AppBarButton Content="Back" Click="NavigateBack" />
-			</CommandBar.PrimaryCommands>
-		</CommandBar>
+		<utu:NavigationBar Content="BlankPage2">
```

Comment:
Isn't the whole point of this sample to show the native frame presenter and the out of the box CommandBar (https://platform.uno/docs/articles/controls/CommandBar.html)

Reply:
Ok, I will update the description of the sample and try to make it work with the command bar. (I used the navigation bar because there were some inconsistencies )

Reply:
https://github.com/unoplatform/uno/issues/13284 made this issue, will wait for a response.

Reply:
reading the description again, I see that I misunderstood a bit. But, I dont think the look of the IOS Commandbar iss acceptable 
![image](https://github.com/unoplatform/Uno.Samples/assets/90481654/4ecfe12c-7625-40bb-967f-a753a0a924b4)
 and on android, you also don't have the native back button even if you put nothing in the primary command on android 
![image](https://github.com/unoplatform/Uno.Samples/assets/90481654/9db5141e-df9f-4fdd-bc14-bc2f808273f2)
that's another bug I guess?

Reply:
Android doesn't need a back button since there's a hardware button (or soft key now that most of those buttons are virtual). I think this is the correct behaviour for now